bundles/org.openhab.binding.solax/README.md

```diff
@@ -71,6 +87,23 @@ In case the parsed information that comes with the binding out of the box differ
 | last-update-time         | DateTime                   | Last time when a call has been made to the inverter                                                                                         |
 | raw-data                 | String                     | The raw data retrieved from inverter in JSON format. (Usable for channels not implemented. Can be consumed with the JSONpath transformation |
 
+### Statistics / Usage related Channels
+
+| Channel                          | Type                       | Description                                               |
+|----------------------------------|----------------------------|-----------------------------------------------------------|
+| power-usage                      | Number:Power               | Current power usage / consumption of the building [W]     |
+| total-energy                     | Number:Power               | Total energy output from the inverter [W]                 |
```

Comment:
Should it be `total-power`? Since it's `Number:Power` in W, i.e. not energy.

Or, if the channel definition is correct:
```xml
<channel id="total-energy" typeId="system.electric-energy">
	<label>Total Energy</label>
	<description>Total energy output from the inverter</description>
</channel>
```

you probably just need to update the Type and Description here.

Reply:
I'm referring to `total-energy`. In the channel definition this is `Number:Energy`. In the README it's `Number:Power`.

Reply:
Sorry. My bad. This is Energy and should be in kWh. I'll fix it.